nss-day-cohort-39/nutshell-veridian-vicuna
# Rendering Events

### Details
- Events now render from the database in a list
- Event list was given a scroll bar
- The top event will always appear larger and with a different background

### How to Test
- Make sure you have object representations of events in your database
- Serve the database and see if events render on page load
